"invalid" aqua.yaml results in SIGSEGV

## aqua version

Please use the latest version.

```console
$ aqua -v
aqua version 1.19.4 (5b32387261f97dd3424bea0bc9fe36820bc210ac)
```

## Environment

* OS (Windows, Linux, macOS, etc): ArchLinux (WSL2)
* CPU Architecture (amd64, arm64, etc): amd64

## Overview

## How to reproduce

* aqua.yaml
* executed command and output

> aqua.yaml should be not partial but complete configuration.
> Please remove unneeded configuration to reproduce the issue.

```yaml
---
# aqua - Declarative CLI Version Manager
# https://aquaproj.github.io/
registries:
  - type: standard
    ref: v3.64.0 # renovate: depName=aquaproj/aqua-registry

packages:
  -
```

```console
$ aqua i -a
```
... or running every other package installed.

## Debug Output

Please set the environment variable `AQUA_LOG_LEVEL=debug` and run aqua and tell us the debug output.

```console
AQUA_LOG_LEVEL=debug aqua i -a
DEBU[0000] install the proxy                             aqua_version=1.19.4 env=linux/amd64 package_name=aqua-proxy package_version=v1.1.2 program=aqua registry=
DEBU[0000] check if aqua-proxy is already installed      aqua_version=1.19.4 env=linux/amd64 package_name=aqua-proxy package_version=v1.1.2 program=aqua registry=
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x38 pc=0x94b2a2]

goroutine 1 [running]:
github.com/aquaproj/aqua/pkg/config-reader.(*ConfigReader).readImports(0xc0001fdac0, {0xc0001987b0, 0x21}, 0xc000227bf0)
        /home/runner/work/aqua/aqua/pkg/config-reader/reader.go:41 +0xa2
github.com/aquaproj/aqua/pkg/config-reader.(*ConfigReader).Read(0xc0001fdac0, {0xc0001987b0, 0x21}, 0x9?)
        /home/runner/work/aqua/aqua/pkg/config-reader/reader.go:32 +0x38a
github.com/aquaproj/aqua/pkg/controller/install.(*Controller).install(0xc000229f10, {0xd5a508, 0xc00011d440}, 0x0?, {0xc0001987b0, 0x21})
        /home/runner/work/aqua/aqua/pkg/controller/install/install.go:93 +0x7f
github.com/aquaproj/aqua/pkg/controller/install.(*Controller).Install(0xc000229f10, {0xd5a508, 0xc00011d440}, 0x1269860?, 0xc000113d40)
        /home/runner/work/aqua/aqua/pkg/controller/install/install.go:65 +0x2fe
github.com/aquaproj/aqua/pkg/cli.(*Runner).installAction(0xc000100b90, 0xc00011da00)
        /home/runner/work/aqua/aqua/pkg/cli/install.go:70 +0x1ea
github.com/urfave/cli/v2.(*Command).Run(0xc00019ed80, 0xc00011d800)
        /home/runner/go/pkg/mod/github.com/urfave/cli/v2@v2.16.3/command.go:173 +0x6ca
github.com/urfave/cli/v2.(*App).RunContext(0xc0002341c0, {0xd5a508?, 0xc00011d440}, {0xc000110060, 0x3, 0x3})
        /home/runner/go/pkg/mod/github.com/urfave/cli/v2@v2.16.3/app.go:383 +0xfde
github.com/aquaproj/aqua/pkg/cli.(*Runner).Run(0xc000100b90, {0xd5a508, 0xc00011d440}, {0xc000110060, 0x3, 0x3})
        /home/runner/work/aqua/aqua/pkg/cli/runner.go:116 +0x16f3
main.core(0xc000229ea0, 0xc000133180)
        /home/runner/work/aqua/aqua/cmd/aqua/main.go:56 +0x277
main.main()
        /home/runner/work/aqua/aqua/cmd/aqua/main.go:30 +0x55
```

## Expected Behaviour

A hint to the user about invalid configuration.

--- a/pkg/config-reader/reader.go
+++ b/pkg/config-reader/reader.go
@@ -38,6 +38,9 @@ func (reader *ConfigReader) Read(configFilePath string, cfg *aqua.Config) error 
 func (reader *ConfigReader) readImports(configFilePath string, cfg *aqua.Config) error {
 	pkgs := []*aqua.Package{}
 	for _, pkg := range cfg.Packages {
+		if pkg == nil {
+			continue
+		}
 		if pkg.Import == "" {
 			pkgs = append(pkgs, pkg)
 			continue

--- a/pkg/config/aqua/config.go
+++ b/pkg/config/aqua/config.go
@@ -74,6 +74,9 @@ func (registries *Registries) UnmarshalYAML(unmarshal func(interface{}) error) e
 	}
 	m := make(map[string]*Registry, len(arr))
 	for _, registry := range arr {
+		if registry == nil {
+			continue
+		}
 		m[registry.Name] = registry
 	}
 	*registries = m

--- a/pkg/config/registry/config.go
+++ b/pkg/config/registry/config.go
@@ -31,6 +31,10 @@ func (pkgInfos *PackageInfos) toMap(logE *logrus.Entry, logLevel logrus.Level) m
 	for _, pkgInfo := range *pkgInfos {
 		logE := logE
 		pkgInfo := pkgInfo
+		if pkgInfo == nil {
+			logE.Log(logLevel, "ignore an empty package")
+			continue
+		}
 		name := pkgInfo.GetName()
 		if name == "" {
 			logE.Log(logLevel, "ignore a package in the registry because the name is empty")

--- a/pkg/install-registry/install.go
+++ b/pkg/install-registry/install.go
@@ -30,7 +30,6 @@ var errMaxParallelismMustBeGreaterThanZero = errors.New("MaxParallelism must be 
 
 func (inst *Installer) InstallRegistries(ctx context.Context, cfg *aqua.Config, cfgFilePath string, logE *logrus.Entry) (map[string]*registry.Config, error) {
 	var wg sync.WaitGroup
-	wg.Add(len(cfg.Registries))
 	var flagMutex sync.Mutex
 	var registriesMutex sync.Mutex
 	var failed bool
@@ -41,6 +40,10 @@ func (inst *Installer) InstallRegistries(ctx context.Context, cfg *aqua.Config, 
 	registryContents := make(map[string]*registry.Config, len(cfg.Registries)+1)
 
 	for _, registry := range cfg.Registries {
+		if registry == nil {
+			continue
+		}
+		wg.Add(1)
 		go func(registry *aqua.Registry) {
 			defer wg.Done()
 			if registry.Name == "" {

--- a/tests/aqua.yaml
+++ b/tests/aqua.yaml
@@ -2,9 +2,11 @@ registries:
 - type: standard
   type: standard # duplicated keys
   ref: v3.64.0 # renovate: depName=aquaproj/aqua-registry
+- null
 
 packages:
 - name: kubernetes-sigs/kind # http package, raw format
   registry: standard
   registry: standard # duplicated keys
   version: v0.16.0 # renovate: depName=kubernetes-sigs/kind
+- null

